src/render.rs

```diff
@@ -141,11 +142,31 @@ impl<'a> Default for MarkdownRenderer<'a> {
 /// let text = "[Rust](https://rust-lang.org/) is an awesome *systems programming* language!";
 /// let rendered = markdown_to_html(text)?;
 /// ```
-pub fn markdown_to_html(text: &str) -> CargoResult<String> {
+fn markdown_to_html(text: &str) -> CargoResult<String> {
     let renderer = MarkdownRenderer::new();
     renderer.to_html(text)
 }
 
+static MARKDOWN_EXTENSIONS: [&'static str; 7] = [
+    ".md",
+    ".markdown",
+    ".mdown",
+    ".mdwn",
+    ".mkd",
+    ".mkdn",
+    ".mkdown",
+];
+
+pub fn readme_to_html(text: &str, filename: &str) -> CargoResult<String> {
```

Comment:
Could you add a doc comment on this function ?

Reply:
Whoops, forgot!